apps/i18n/common/en_us.json

```diff
@@ -348,6 +348,7 @@
   "chooseSection": "Choose your section",
   "chooseSectionsPrompt": "Which section(s) do you want to assign \"{assignmentName}\" to?",
   "chooseSectionsDirections": "When you assign a curriculum to a section, that curriculum will be the first thing students see upon signing in. Changing the assigned curriculum will never affect students progress on other curriculum.",
+  "chooseSectionsDirectionsOnCatalog": "When you assign a curriculum to a section, that curriculum will be the first thing students see upon signing in. Changing the assigned curriculum will never affect students progress on other curriculum. Note: The most recent recommended version of the curriculum will be assigned to your course when you assign from the Curriculum Catalog.",
```

Comment:
My bad. Can you change "assigned to your course" to "assigned to your section"

Reply:
Oh good catch –– yes, will update!